Gemfile

```diff
@@ -77,6 +77,7 @@ gem 'riiif', '~> 2.0'
 gem 'sitemap_generator'
 
 gem 'blacklight-gallery', '>= 0.3.0'
+gem 'blacklight-hierarchy', github: 'sul-dlss/blacklight-hierarchy'
```

Comment:
We need to cut a release that is BL 7 compatible.  Happy to do that as part of this.